### ISSUE TITLE
Missing Obb Folder Has Been Finally Found!

### ISSUE THREAD

**Issue by a7md8762** [niizam/Genymotion_A11_libhoudini#7]
I Have Finally Found Where The obb Folder For Those Who Got The "Missing obb Folder" After Installing The Lib!

It's In: `/data/media/0/android`

**Comment by prob4bly-anon** [comment]
> I Have Finally Found Where The Obb Folder For Those Who Got The "Missing Obb Folder" Problem After Installing The Library!
> 
> It's In: `/data/media/0/android` (You Have To Be Rooted)
> 
> But There Is Something, You Can't Let The Game Access That Obb Folder, So The Game Won't Run, If Anyone Has A Way To Make The Game Access To The New Obb Folder (`/data/media/0/android`), I Would Appreciate It!

@a7md8762 Have you tried creating symlink for it?

**Comment by a7md8762** [comment]
> > I Have Finally Found Where The Obb Folder For Those Who Got The "Missing Obb Folder" Problem After Installing The Library!
> > It's In: `/data/media/0/android` (You Have To Be Rooted)
> > But There Is Something, You Can't Let The Game Access That Obb Folder, So The Game Won't Run, If Anyone Has A Way To Make The Game Access To The New Obb Folder (`/data/media/0/android`), I Would Appreciate It!
> 
> @a7md8762 Have you tried creating symlink for it?

No, I didn't unfortunately, But can you even symlink it? If so, that would fix must of the problems we are facing.
Also, I think I have deleted everything related to Genymotion as I moved to using emulators on Windows.
